tests/unit/peapods/runtimes/asyncio/rest/test_models.py

```diff
@@ -73,45 +69,50 @@ def test_all_fields_in_document_proto():
     ]:
         assert i in document_proto_properties
 
-    document_proto_properties_alias = PROTO_TO_PYDANTIC_MODELS[
-        'DocumentProto'
-    ]().schema()['definitions']['DocumentProto']['properties']
+    document_proto_properties_alias = PROTO_TO_PYDANTIC_MODELS.DocumentProto().schema()[
+        'definitions'
+    ]['DocumentProto']['properties']
     for i in ['contentHash', 'levelName', 'parentId', 'mimeType']:
         assert i in document_proto_properties_alias
 
 
 def test_oneof():
```

Comment:
Testing a case with oneof types in different levels would be nice.
For instance testing a Document with some content and an embedding

Reply:
Good point. I'll add this in a later pr.